CRM/Campaign/Form/Search/Campaign.php

```diff
@@ -78,11 +78,8 @@ public function buildQuickForm() {
     //campaign description.
     $this->add('text', 'description', ts('Description'), $attributes['description']);
 
-    //campaign start date.
-    $this->addDate('start_date', ts('From'), FALSE, ['formatType' => 'searchDate']);
-
-    //campaign end date.
-    $this->addDate('end_date', ts('To'), FALSE, ['formatType' => 'searchDate']);
+    $this->add('datepicker', 'start_date', ts('From'), [], FALSE, ['time' => FALSE]);
```

Comment:
@eileenmcnaughton this works but just wondering if we should put something like Start Date From as the label and a la the end_date as at the moment its not obvious what field your filtering

Reply:
@seamuslee001 OK - I checked the xm & it uses Campaign Start Date so using that - I can't be bothered switching to addField for this last obscure form but for the search range ones on the search forms we should do it properly (like we did for activity & grant)